docs/PROJECTS_API.md

```diff
@@ -195,4 +196,30 @@ Renderforest.renderProject(4120385, { quality: 360, watermark: 'https://example.
 
 [See example](https://github.com/renderforest/renderforest-sdk-node/blob/master/samples/projects/render-project.js)
 
+
+### Get rendering status
+
+Our rendering status method uses user's defined `callback` to manage rendering status percentage 
+and uses error first callback pattern. If you want to unsubscribe from getting rendering status then simply call 
+`unsubscribe` (`getRenderingStatus` method returns function to unsubscribe from getting rendering status).
+
+```js
+const RenderforestClient = require('@renderforest/sdk-node')
+
+const Renderforest = new RenderforestClient({ signKey: '<signKey>', clientId: -1 })
+
+Renderforest.renderProject(15431416, { quality: 720 })
+  .then(() => {
+    const unsubscribe = Renderforest.getRenderingStatus((error, percentage) => {
+      if (error) {
+        console.log(error)
+      }
+      // take percentage from here
+      console.log(percentage)
+      // if you want in unsubscribe in some case, then simply call unsubscribe
```

Comment:
// if you want in unsubscribe (before rendering finishes) for some reason, then simply call unsubscribe